src/main/java/org/spongepowered/common/mixin/optimization/pathfinding/PathNavigateMixin_ChunkLoadOptimizations.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.mixin.optimization.pathfinding;
+
+import net.minecraft.pathfinding.PathNavigate;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(value = PathNavigate.class, priority = 1500)
+public class PathNavigateMixin_ChunkLoadOptimizations {
+
+    @Shadow protected World world;
+
+    /**
+     * Check if a chunk is loaded before attempting to check the state of the block
+     * return false if the chunk is not loaded
+     * @author Clienthax
+     */
+    @Overwrite
+    public boolean canEntityStandOnPos(BlockPos pos) {
```

Comment:
No, an overwrite is not the thing to do here. You can _easily_ make this a cancellable injection at the head of the method instead. Overwrites should be used as a last resort.

Also, if `pos` is loaded, `pos.down()` will be loaded as it'll be in the same chunk, right? You can drop the `.down()` call in the injection to remove a method call.